kubernetes/samples/scripts/create-rcu-schema/create-rcu-schema.sh

```diff
@@ -21,7 +21,7 @@ function usage {
   exit $1
 }
 
-while getopts ":h:s:d:p:" opt; do
+while getopts ":h:s:d:p:i:" opt; do
```

Comment:
should be doc for this new `i` option you are adding

Reply:
Actually it was a typo in the existing script for image input parameter.  The description of the parameter is already there.

<bash>./create-rcu-schema.sh 
create-rcu-schema.sh: -s <schemaPrefix> must be specified.
usage: create-rcu-schema.sh -s <schemaPrefix> -d <dburl> -i <image> -p <docker-store> [-h]
  -s RCU Schema Prefix (needed)
  -d RCU Oracle Database URL (optional) 
      (default: oracle-db.default.svc.cluster.local:1521/devpdb.k8s) 
  -p FMW Infrastructure ImagePull Secret (optional) 
      (default: docker-store) 
  -i FMW Infrastructure Image (optional) 
      (default: container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.3) 
  -h Help